.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+npm-debug.log
```

Comment:
node_modules makes sense, but take out npm-debug.log: the debug log gets cleared by npm automatically if it's generated by bad code that is subsequently fixed, so the only reason this file would be in a commit is if you're pushing it up while the code doesn't work, which means you shouldn't be pushing up yet.